packages/next-templates/src/app/card-component/card/page.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import { Heading } from '@utrecht/component-library-react';
+import clsx from 'clsx';
+import Image from 'next/image';
+import React, { FC, HTMLAttributes } from 'react';
+import './index.style.css';
+
+interface CardListItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'children'> {
+  headerlevel: number;
+  title?: string;
+  preheading?: string;
+  children?: string;
+  image?: {
+    url: string;
+    alt: string;
+  };
+  link: {
+    href: string;
+  };
+
+  // do we want an article around the card content. Default is no
+  useArticle?: boolean | false;
```

Comment:
Perhaps `role` would be a good name for this API. How about: `cardRole?: "article" | string;`

Then you can do:

```jsx
let card = <div>...</div>;

if (cardRole === "article") {
  card = <article>{card}</article>
}
```